src/test/fetchCoin.test.js

```diff
@@ -0,0 +1,8 @@
+import fetchSingleCoin from '../services/fetchCoin';
+
+describe('fetchSingleCoin', () => {
+  it('should fetch a single coin data', async () => {
+    const coinData = await fetchSingleCoin('bitcoin');
+    expect(coinData).toBeDefined();
+  });
+});
```

Comment:
- Hey, please you should add more test to test your components, you've only added just two tests. Having added the tests, your test is failing, please, kindly fix it. see the images below:

- According to the project requirement, your app should have unit tests for several components of the app and some pure functions in Redux code.


![image](https://github.com/Peter-Kibet/metrics-app-react-capstone/assets/59397159/cc133d19-8aca-4d69-b51d-c19344ced4eb)